units/en/unit0/introduction.mdx

```diff
@@ -0,0 +1,126 @@
+
+Don’t forget to **<a href="http://eepurl.com/ic5ZUD">sign up to the course</a>** (we are collecting your email to be able to **send you the links when each Unit is published and give you information about the challenges and updates).**
+
+Sign up  👉 <a href="http://eepurl.com/ic5ZUD">here</a>
+
+
+## What does the course look like? [[course-look-like]]
+The course is composed of:
+
+- *A theory part*: where you learn a **concept in theory (article)**.
+- *A hands-on*: where you’ll learn **to use famous Deep RL libraries** to train your agents in unique environments. These hands-on will be **Google Colab notebooks but also tutorial videos**.
+- *Challenges*: such AI vs. AI and leaderboard.
+
+
+## Two paths: choose your own adventure [[two-paths]]
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit0/two-paths.jpg" alt="Two paths" width="100%"/>
+
+You can choose to follow this course either:
+
+- *To get a certificate of completion*: you need to complete 80% of the assignments before the end of March 2023.
+- *As a simple audit*: you can participate in all challenges and do assignments if you want, but you have no deadlines.
+
+Whatever path you choose, we advise you **to follow the recommended pace to enjoy the course and challenges with your fellow classmates.**
+You don't need to tell us which path you choose. At the end of March, when we verify the assignments **if you get more than 80% of the assignments done, you'll get a certificate.**
+
+
+
+## How to get most of the course? [[advice]]
+
+To get most of the course, we have some advice:
+
+1. <a href="https://discord.gg/ydHrjt3WP5">Join or create study groups in Discord </a>: studying in groups is always easier. To do that, you need to join our discord server.
+2. **Do the quizzes and assignments**: the best way to learn is to do and test yourself.
+3. **Define a schedule to stay in sync**: you can use our recommended pace schedule below or create yours.
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit0/advice.jpg" alt="Course advice" width="100%"/>
+
+## What tools do I need? [[tools]]
+
+You need only 3 things:
+
+- *A computer* with an internet connection.
+- *Google Colab (free version)*: most of our hands-on will use Google Colab, the **free version is enough.**
+- A *Hugging Face Account*: to push and load models. If you don’t have an account yet you can create one here (it’s free).
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit0/tools.jpg" alt="Course tools needed" width="100%"/>
+
+
+## What is the recommended pace? [[recommended-pace]]
+
+We defined a planning that you can follow to keep up the pace of the course.
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit0/pace1.jpg" alt="Course advice" width="100%"/>
```

Comment:
Maybe it's too complex to change it now, but maybe consider adding one or two gap weeks in early january.

Reply:
✔️ I can change the recommended pace I put it on my list